modules/dind/coverage.out

```diff
@@ -0,0 +1,26 @@
+mode: atomic
```

Comment:
suggestion: let's remove the coverage file from the PR. We probably need to ignore on our side, I'll do it in a separate PR